components/http_foundation/introduction.rst

```diff
@@ -344,7 +344,37 @@ attribute::
 The
 :method:`Symfony\\Component\\HttpFoundation\\ResponseHeaderBag::setCookie`
 method takes an instance of
-:class:`Symfony\\Component\\HttpFoundation\\Cookie` as an argument.
+:class:`Symfony\\Component\\HttpFoundation\\Cookie` as an argument. This class
+takes eight arguments in its constructor:
+
+``$name``
+    **type**: ``string`` **default**: none (this argument is mandatory)
+    The name of the cookie.
+``$value``
+    **type**: ``string`` **default**: ``null``
+    The value stored in the cookie.
+``$expire``
+    **type**: ``int``|``string``|``DateTime``|``DateTimeInterface`` **default**: 0
+    The time the cookie expires. This value can be set as a timestamp integer,
+    as a ``strtotime()`` valid date string (e.g. ``+1 week``), as a ``DateTime``
```

Comment:
For convenience, you could link the strtotime docs in here. This makes it a lot easier to find the possible values for those who are unfamiliar with it